src/components/HeaderForm.js

```diff
@@ -52,7 +52,7 @@ const EMPTY_STATE = {
   gamFile: undefined,
   bedFile: undefined,
   dataPath: undefined,
-  region: undefined,
+  region: "none",
```

Comment:
Does this have to be "none" and not, say, an empty string?

Why does this part help solve the problem? Or was it adding `...EMPTY_STATE,` to the code to swap over to mounted files that did it? Or a combination of the two?